packages/util/src/styles.ts

```diff
@@ -8,7 +8,7 @@ export function stylesFor(selectorKey: string, props: Props) {
   )
 }
 
-interface Props {
+export interface Props {
```

Comment:
I'd like this to be a more specific name, since `util` pkg is a junk drawer and we are exporting tons of named stuff in the same namespace.

```suggestion
export interface StylesForProps {
```